modules/ROOT/pages/dataweave-cookbook-include-xml-namespaces.adoc

```diff
@@ -51,6 +51,24 @@ include::partial$cookbook-dw/include-xml-namespaces-ex2/transform.dwl[]
 include::partial$cookbook-dw/include-xml-namespaces-ex2/out.xml[]
 ----
 
+== Use XML Namespace with Hyphen Character
+
+In some projects, you might require to use XML namespaces with special characters, such as hyphen `-`. The following DataWeave script shows that to use `-` in the namespace, you must define the namespace as an object.
```

Comment:
I'd suggest a rewording of this whole description for better clarity, something like this: 

> The following DataWeave script shows you how to define an XML namespace as an object, which enables you to use special characters such as hyphen (`-`) in the namespace definition: